Redis Express App

akshat99812 opened the pull request:
This PR is related to the following issue:
https://github.com/keploy/keploy/issues/2296

Request for Review: @khareyash05  @Sonichigo  – Kindly take a moment to review this PR. Your feedback will be highly appreciated.

khareyash05 commented on .DS_Store:
Please remove this file, not needed

khareyash05 commented on redis-express/README.md:
````diff
@@ -0,0 +1,109 @@
+
+```bash
+ npm install
+ npm run build
````
In case of running application natively, we will have to change the host to localhost in `index.ts`

khareyash05 commented on redis-express/README.md:
````diff
@@ -0,0 +1,109 @@
+```
+
+```bash
+root@MacBook-Air express-redis % npm run build
````
can remove this code block

akshat99812 commented on redis-express/README.md:
````diff
@@ -0,0 +1,109 @@
+
+```bash
+keploy record -c "npm run start"
+```
````
added 5 curl commands for testing

akshat99812 commented on redis-express/README.md:
````diff
@@ -0,0 +1,109 @@
+```
+
+```bash
+root@MacBook-Air express-redis % npm run start
````
done

khareyash05 commented on .DS_Store:
Please remove this file

akshat99812 replied:
file removed from server directory

khareyash05 commented on redis-express/README.md:
```diff
@@ -0,0 +1,164 @@
+
+In case of running application natively, we will have to change the host to localhost in index.ts
+
+
```
Please check the Preview of this code, looks disoriented 
<img width="937" alt="Screenshot 2024-10-01 at 18 34 05" src="https://github.com/user-attachments/assets/80d68714-6b04-47d1-a6d5-d4e8f730fdcc">

khareyash05 commented on redis-express/README.md:
```diff
@@ -0,0 +1,164 @@
+
+
+
+The above command will start recording the API calls made to the application and will generate a test case in the `testcases/` directory.
```
`testcases` folder? should be changed to `keploy`, please look into the docs

khareyash05 commented on redis-express/README.md:
````diff
@@ -0,0 +1,164 @@
+
+```
+
+Voila! 🎉 You have successfully tested the application using Keploy. Keploy also generates coverage reports for the test-suites.
````
<img width="937" alt="Screenshot 2024-10-01 at 18 35 27" src="https://github.com/user-attachments/assets/c4454a0c-1234-4e49-8036-727a0606ba75">
Please format,

akshat99812 replied:
file formatted properly

khareyash05 reviewed (changes requested):
Also since you have a docker setup, please mention the docker setup instructions as well.

akshat99812 commented:
@khareyash05 I have made the required changes in the readme file , if there need be any changes let me know

khareyash05 reviewed (changes requested):
Add Documentation to setup app with Docker as well. Also as addressed before, please edit the markdown using a markdown editor

khareyash05 commented:
Also please attach the screenshots of the markdown after your commit, will be easier to check

akshat99812 commented:
<img width="1279" alt="Screenshot 2024-10-16 at 4 48 02 PM" src="https://github.com/user-attachments/assets/71d92a18-a6df-42c2-a1fd-5e93e9d07dad">
<img width="1279" alt="Screenshot 2024-10-16 at 4 48 14 PM" src="https://github.com/user-attachments/assets/ffb69248-6e4f-4a23-ba3d-c26a73f41811">
<img width="1279" alt="Screenshot 2024-10-16 at 4 47 54 PM" src="https://github.com/user-attachments/assets/ed2ef3f0-73d0-42f8-952d-9ed53c2f7850">

Added instructions for docker container and cleaned the readme file using online editor.

khareyash05 reviewed (approved):
LGTM!
